### PR TITLE
Mx 15407 sovereign proxy scripts

### PR DESCRIPTION
**Pull request by axenteoctavian**
## Reasoning behind the pull request
- 
- 
- 
  
## Proposed changes
- sovereign local testnet: proxy deploy script update
- 
- 

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: scripts/testnet/include/tools.sh [hunk 1]
```diff
@@ -5,7 +5,10 @@ startProxy() {
   setTerminalLayout "even-vertical"
 
   setWorkdirForNextCommands "$TESTNETDIR/proxy"
-  runCommandInTerminal "./proxy" $1
+  if [ "$SOVEREIGN_DEPLOY" -eq 1 ]; then
+      ARGS="--sovereign-config"
+  fi
+  runCommandInTerminal "./proxy $ARGS" $1
```
Comment (mariusmihaic) [inline scripts/testnet/include/tools.sh:11]: `ARGS` is undefined unless `SOVEREIGN_DEPLOY` is true.

**Review by mariusmihaic — CHANGES_REQUESTED** [review]

File: scripts/testnet/include/tools.sh (hunk 1 above)
Comment (sasurobert) [reply scripts/testnet/include/tools.sh:11]: +1

**Review by sasurobert — COMMENTED** [review]
Only what Marius commented.

File: scripts/testnet/include/tools.sh [hunk 2]
```diff
@@ -6,7 +6,9 @@ startProxy() {
 
   setWorkdirForNextCommands "$TESTNETDIR/proxy"
   if [ "$SOVEREIGN_DEPLOY" -eq 1 ]; then
-      ARGS="--sovereign-config"
+      ARGS="--sovereign"
+  else
```
Comment (mariusmihaic) [inline scripts/testnet/include/tools.sh:10]: I think we should define it **above the first if** `ARGS=""`
Then only change it `if [ "$SOVEREIGN_DEPLOY" -eq 1 ]`(and remove else)

**Review by mariusmihaic — APPROVED** [review]

**Pull request merged**